components/cli/pkg/constants/const.go

```diff
@@ -56,10 +56,10 @@ const CELLERY_MANAGE_START = "start"
 const CELLERY_MANAGE_CLEANUP = "cleanup"
 
 const AWS_S3_BUCKET = "cellery-runtime-installation"
-const AWS_S3_ITEM_VM_COMPLETE = "cellery-runtime-complete-0.2.0.tar.gz"
-const AWS_S3_ITEM_VM_MINIMAL = "cellery-runtime-basic-0.2.0.tar.gz"
-const AWS_S3_ITEM_CONFIG_COMPLETE = "config-cellery-runtime-complete-0.2.0"
-const AWS_S3_ITEM_CONFIG_MINIMAL = "config-cellery-runtime-basic-0.2.0"
+const AWS_S3_ITEM_VM_COMPLETE = "cellery-runtime-complete-0.3.0-SNAPSHOT.tar.gz"
```

Comment:
Shouldn't the version be 0.2.1-SNAPSHOT ? (Just wondering as we are doing a 0.2.1 release next)

Reply:
+1, fixed in 7b8522340fdd8ea64697a5a764d540d614151401